<launch>
    <node pkg="bot_sim" type="dbscan_bfs" name="dbscan_bfs" output="screen">
        <param name="epsilon" type="double" value="0.2" />
        <param name="minPts" type="int" value="7" />
        <param name="dfs_decrease" type="int" value="5" />
        <param name="dfs_threshold" type="int" value="65" />
    </node>
</launch>
